src/ducks/groups/helpers.spec.js

```diff
@@ -158,3 +159,21 @@ describe('translateAndSortGroups', () => {
     expect(setup(groups)).toEqual(expected)
   })
 })
+
+describe('renamed group', () => {
+  it('should work', () => {
+    const group = {
+      label: 'My group'
+    }
+    const autogroup = {
+      accountType: 'checkings'
+    }
+    expect(renamedGroup(group, 'My super group')).toEqual({
+      label: 'My super group'
+    })
+    expect(renamedGroup(autogroup, 'My super group')).toEqual({
+      label: 'My super group',
+      accountType: null
+    })
+  })
```

Comment:
I suggest

```js
describe('when the given group has an accountType', () => {
  it('should not set accountType to null', () => {
    const group = { accountType: 'checkings' }

    expect(renamedGroup(group, 'My super group')).toEqual({
      label: 'My super group',
      accountType: null
    })
  })
})

describe('when the given group does not have an accountType', () => {
  it('should not set accountType to null', () => {
    const group = { label: 'My group' }

    expect(renamedGroup(group, 'My super group')).toEqual({
      label: 'My super group'
    })
  })
})
```

I agree this is more verbose, but I think having everything under a `should work` test case is not descriptive enough.

Reply:
ok, thanks for the code I copy pasted it.